pkgs/by-name/br/brlaser/package.nix

```diff
@@ -18,6 +19,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-a+TjLmjqBz0b7v6kW1uxh4BGzrYOQ8aMdVo4orZeMT4=";
   };
 
+  patches = [
```

Comment:
There is no need for `patches` here as you can just override the variable as the error suggests:

```diff
diff --git a/pkgs/by-name/br/brlaser/package.nix b/pkgs/by-name/br/brlaser/package.nix
index 2cc36dd9d4ed..ceabcc664bdf 100644
--- a/pkgs/by-name/br/brlaser/package.nix
+++ b/pkgs/by-name/br/brlaser/package.nix
@@ -31,6 +31,7 @@ stdenv.mkDerivation rec {
   cmakeFlags = [
     "-DCUPS_SERVER_BIN=lib/cups"
     "-DCUPS_DATA_DIR=share/cups"
+    "-DCMAKE_POLICY_VERSION_MINIMUM=3.5"
   ];
 
   meta = {
```